src/components/ErrorMessage.tsx

```diff
@@ -14,7 +15,21 @@ const ErrorMessage: FC = () => {
     <TransitionGroup>
       {value ? (
         <CSSTransition key={0} nodeRef={errorMessageRef} timeout={200} classNames='fade'>
-          <div ref={errorMessageRef} className='error-message'>
+          <div
+            ref={errorMessageRef}
+            className={css({
+              position: 'fixed',
+              top: 'safe-area-top',
+              left: '0',
+              right: '0',
+              padding: '5px 25px 5px 5px',
+              backgroundColor: '#c23',
+              color: 'white',
```

Comment:
Do you want to convert colors to design tokens as you go, or later on?

Reply:
I'd like to do it as I go, however for this case, it looks like the text will have to be white anyway, to be seen against a red background.
<img width="1725" alt="Screenshot 2024-08-28 at 4 06 45 PM" src="https://github.com/user-attachments/assets/72cf4a83-dab8-428b-9ee1-4102f628715f">